src/foam/dao/AbstractJDAO.java

```diff
@@ -79,18 +79,19 @@ protected void loadJournal()
         // remove first two characters and last character
         line = line.trim().substring(2, line.trim().length() - 1);
 
-        FObject object = parser.parseString(line);
-        if ( object == null ) {
-          System.err.println(getParsingErrorMessage(line) + ", source: " + line);
-          break;
```

Comment:
@kgrgreer @jlhughes This break statement was causing a journal file to stop being parsed